src/wallet/rpc.rs

```diff
@@ -126,7 +126,7 @@ impl Wallet {
                 .unwrap_or(0)
                 .max(self.store.wallet_birthday.unwrap_or(0));
             let node_synced = self.rpc.get_block_count()?;
-            log::info!(
+            log::debug!(
```

Comment:
The sync will always be upto current tip height. We added this to see if the sync is working properly or not. Users won't find this useful for anything, and if it happens a lot, it will unnecessarily clutter the logs. In general all other internal module level logs should be debug. 

If we need to see it for debugging, we can ask the user to log in with debug.